containers/java-postgres/test-project/src/main/java/com/mycompany/app/App.java

```diff
@@ -0,0 +1,34 @@
+/*-------------------------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+ *-------------------------------------------------------------------------------------------------------------*/
+
+package com.mycompany.app;
+
+import static javax.xml.XMLConstants.XML_NS_PREFIX;
+import java.io.*;
+import java.net.*;
+
+
+
+public class App 
+{
+    public static void main( String[] args )
+    {
+        System.out.println( "Hello Remote World!" );
+        System.out.println("The XML namespace prefix is: " + XML_NS_PREFIX);
+
+    }
+
+    public static boolean pingAddress(String ipAddress) throws UnknownHostException, IOException
+    {
+        InetAddress postgresAddress = InetAddress.getByName(ipAddress);
```

Comment:
why is this in `App.java` if it's not invoked by the app itself?

Reply:
We'll be changing App.java to add new tests/capability, so that will cover this

Reply:
Done!